webapp/containers/add_users_to_team.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {getProfilesNotInTeam} from 'mattermost-redux/actions/users';
+
+import AddUsersToTeam from 'components/add_users_to_team.jsx';
+
+function makeMapStateToProps() {
```

Comment:
why are we wrapping the mapStateToProps function in another function here? it doesn't seen necessary